docs/source/config/callbacks.rst

```diff
@@ -64,16 +71,32 @@ skipping the history/display mechanisms, in which cases ``pre_run_cell`` will no
 post_run_cell
 -------------
 
-``post_run_cell`` runs after interactive execution (e.g. a cell in a notebook).
-It can be used to cleanup or notify or perform operations on any side effects produced during execution.
-For instance, the inline matplotlib backend uses this event to display any figures created but not explicitly displayed during the course of the cell.
-
+``post_run_cell`` runs after successful interactive execution (e.g. a cell in a
+notebook, but, for example, not when a ``SyntaxError`` was raised).
```

Comment:
If we leave the `post_` events as they are (I think we should consider changing them), this documentation needs to be a bit more precise. They still fire after a normal error (e.g. from `1/0`), it's only if preprocessing/parsing/compiling the code fails, so we don't actually execute it, that they don't fire.

Reply:
Sure, will update this in the next commit...